apps/web-giddh/src/app/invoice/preview/models/bulk-export-modal/bulk-export.component.html

```diff
@@ -0,0 +1,111 @@
+                        <label for="radio-3" class="radio-custom-label">Ascending</label>
+                    </span>
+                    <span class="pd0 mrR3">
+                        <input id="radio-4" name="orderType" type="radio" class="radio-custom cp ml0">
+                        <label for="radio-4" class="radio-custom-label">Descending</label>
+                    </span>
+                </div>
+            </div>
+        </div> -->
+        <div class="row mb-15">
+            <div class="col-sm-4">
+                <label>Select Copy</label>
+            </div>
+            <div class="col-sm-8">
+                <sh-select [options]="downloadCopyOptions" placeholder="Select Order" [multiple]="true" name="downloadCopy" [(ngModel)]="copyTypes" [isFilterEnabled]="true" [useInBuiltFilterForIOptionTypeItems]="true">
+                    <ng-template #optionTemplate let-option="option">
+                        <ng-container>
+                            <a href="javascript:void(0)" class="list-item d-flex">
+                                <span class="mwauto"><input type="checkbox" [checked]="copyTypes.includes(option.value)" [value]="option.value"></span>&nbsp;
+                                <span>
+                                    <div class="item">{{ option.label }}</div>
+                                </span>
+                            </a>
+                        </ng-container>
+                    </ng-template>
+                </sh-select>
+            </div>
+        </div>
+        <!-- <div class="row mb-15">
+            <div class="col-sm-4 d-flex">
+                <input type="checkbox" class="show-on-mobile mr-3">
+                <label>Attachment</label>
+            </div>
+            <div class="col-sm-8 hide-on-mobile">
+                <div class="toggle d-inline-block" data-key="disabled">
+                    <input type="checkbox">
+
+                </div>
+            </div>
+        </div> -->
+        <div class="row mb-15">
+            <div class="col-sm-12">
+            <button class="btn btn-success" [disabled]="!copyTypes || copyTypes.length === 0 || isLoading" (click)="exportVouchers(false)">Download</button>
+                <!-- <button class="btn btn-success">Print</button> -->
+            </div>
+        </div>
+        OR
+        <br>
+        <br>
+        <div class="row mb-15">
+            <div class="col-sm-12">
+                <label class="mb-1">You can add multiple id's comma separated</label>
+            </div>
+            <div class="col-sm-12">
+                <!-- Email id could be mulitiple   -->
+                <textarea type="text" placeholder="Recipient Email Id's" class="form-control mb-15 mulitiple-email" name="recipients" [(ngModel)]="recipients"></textarea>
+                <button class="btn btn-success" [disabled]="!recipients || copyTypes.length === 0 || isLoading" (click)="exportVouchers(true)">Send Email</button>
```

Comment:
Please check if `recipients` needs to come here
```suggestion
                <button class="btn btn-success" [disabled]="!recipients || recipients.length === 0 || isLoading" (click)="exportVouchers(true)">Send Email</button>
```